mungegithub/mungers/release-note-label_test.go

```diff
@@ -249,6 +249,11 @@ func TestGetReleaseNote(t *testing.T) {
 			expectedReleaseNoteVariable: releaseNoteNone,
 		},
 		{
+			body:                        "**Release note**:\n\n<other unimportant information> ```NONE```",
```

Comment:
also, check out the PR template https://github.com/kubernetes/kubernetes/edit/master/.github/PULL_REQUEST_TEMPLATE.md 

The goal of the regex is to match what's there explicitly.